transport/transport.go

```diff
@@ -303,6 +303,24 @@ func StreamFromContext(ctx context.Context) (s *Stream, ok bool) {
 	return
 }
 
+// Peer connection info
+type Peer struct {
```

Comment:
"transport" is an internal package which is supposedly used by grpc internal packages only (e.g., grpc package) instead of the library users. Is it possible to put "Peer" into grpc package?